recipes/reago/meta.yaml

```diff
@@ -13,9 +13,19 @@ requirements:
   build:
     - python
     - setuptools
+    - os
```

Comment:
No. This is not working. Please test this locally. os, sys, time, operator are all in the standard lib of pyhon. Networkx is a module that needs to be installed.